Krytskaya.feature

```diff
@@ -0,0 +1,53 @@
+Feature: 1.1. Contacts
+
+Background:
+    Given user clicked on Notifications "Contacts" 
+
+Scenario Outline: 1.1.1 All external links should be active and right
+    Given user scrolled Contacts to bottom app bars
+    When user clicked on "<Button>" icon
+    Then open a new window in browser 
+    And open an external link <Go to>:
+        | Button   | Go to                                     |
+        | site     | https://www.quantori.com/                 |
+        | mail to  | career@quantori.com                       |

```

Comment:
When we try to open career@quantori.com we see the window for choise the app for sending e-mail. I think this result should be described in the Scenario outline.

Reply:
It shoud be described in separate Scenario outline?
Ok

Reply:
I think it would be better, because it is not exactly open external link, it is a dialog window for sending e-mail